packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx

```diff
@@ -209,7 +210,6 @@ export const SettingsObjectFieldEdit = () => {
             <Section>
               <H2Title title="Values" description="The values of this field" />
               <SettingsDataModelFieldSettingsFormCard
-                disableCurrencyForm
```

Comment:
what was that used for? Nice if we can remove it

Reply:
This was used for disabling currency selector after form creation. I renamed `disableCurrencyForm` prop to `isCreatingField`, because it might be used with other form options than currency too.

Reply:
In `SettingsObjectNewFieldStep2.tsx`, `isCreatingField` is passed to this `SettingsDataModelFieldSettingsFormCard` component.